site/content/how-to/monitoring/tracing.md

```diff
@@ -0,0 +1,335 @@
+---
+title: "Tracing"
+description: "Learn how to configure tracing in NGINX Gateway Fabric."
+weight: 200
+toc: true
+docs: "DOCS-000"
+---
+
+{{<custom-styles>}}
```

Comment:
```suggestion
Learn how to configure tracing in NGINX Gateway Fabric.
```

Shifted description from the frontmatter: the custom styles shortcode is also no longer necessary, as they're the defaults in the theme now.